index.d.ts

```diff
@@ -1,16 +1,12 @@
 // import vue.js typings
 // import Vue from 'vue';
 import { Vue, VueConstructor } from 'vue/types/vue'
-import { Page, NavigationEntry } from 'tns-core-modules/ui/frame/frame'
+import { Page, NavigationEntry, Size } from 'tns-core-modules/ui/frame/frame'
 import { View } from 'tns-core-modules/ui/core/view'
 
-export interface NavigationEntryVue extends NavigationEntry {
```

Comment:
Your branch looks to be out of date, as you've removed changes to add the enhanced type `NavigationEntryVue` which was added to address this issue: #409

Reply:
I have updated the pr, please review @jjstrat3